assets/training/finetune_acft_image/environments/acpt_image_framework_selector/asset.yaml

```diff
@@ -1,5 +1,5 @@
 name: acpt-automl-image-framework-selector-gpu
-version: "7"
+version: "8"
```

Comment:
Why do we need to bump up environment versions?
As a sidenote, these environments are already released with these versions with ubuntu vulnerability related fix. If require version bumping, we need to upgrade it to version "9"